reposerver/repository/repository_test.go

```diff
@@ -279,29 +279,6 @@ func TestGenerateKsonnetManifest(t *testing.T) {
 	assert.Equal(t, "https://kubernetes.default.svc", res.Server)
 }
 
-func TestGenerateHelmChartWithDependencies(t *testing.T) {
-	service := newService("../..")
-
-	cleanup := func() {
-		_ = os.Remove(filepath.Join("../../util/helm/testdata/helm2-dependency", helmDepUpMarkerFile))
```

Comment:
Instead of deleting this test entirely, can you just point it to `../../util/helm/testdata/dependency`?

Reply:
Yes, and i've added back two values files required for a test